assessment.md

```diff
@@ -13,81 +13,512 @@
+  folders, branches, forks, etc.
 * Mitigating software risk
+  * N = No procedure is followed, the master branch is broken, or files are corrupted
+  * I = The engineer avoids causing damage to the work of others, but may not produce
+  any safely compatible source code
+  * A = Major issues are avoided and a few small complications may occur, none that
+  ruin the flow of work for the team
+  * G = Almost no problems occur while working with code or branches, does not slow
+  down productivity for self or others
+  * E = All interactions with new and existing code are free from complications
+  or other issues, everything works smoothly and completely free of errors
 
 ## Professional Skills
 
 ### Individual
 
 * Continuous learning about
   * Python software development
-  * Project management with GitHub
+    * N = A lack of new knowledge and/or contributions
+        to any project
+    * I = Incomplete knowledge or minuscule contributions
+        to projects
+    * A = An average amount of knowledge and contributions
+        to each project
+    * G = An above average amount of knowledge and contributions
+        to each project
+    * E = Going above and beyond the expectations and contributions
+        of others
+  * Project management with GitHub, specifically the Flow Model
+    * N = A complete disregard for the model, especially multiple
+        instances of overriding the master branch
+    * I = Occasional disregard for the model including overriding
+        the master branch
+    * A = Almost always adhering to model, but never overriding
+        the master branch
+    * G = Making only minor mistakes while trying to follow the model
+        and providing a positive influence on other aspects such as issues
+        and communication
+    * E = A lack of mistakes in addition to emerging as a leader who
+      provides support to others who may be struggling with project
+      management
   * Continuous integration with Travis
-  * Foundations of software engineering
+    * N = Failing to enable Travis at all or not passing any requirements
+      on any assignment
+    * I = Enabling Travis, but failing the majority of checks
+    * A = Enabling Travis, and passing a majority of Travis checks
+    * G = Enabling Travis, and passing all of the Travis checks
+    * E = Enabling Travis, passing all checks, and assisting others who are
+      struggling or improving issues with Travis if found
+  * Foundations of software engineering practices
+    * N = Working separate from a team with no communication
+    * I = Working with a team a small amount of the time with no communication
+    * A = Working well with a team half of the time with little communication
+    * G = Working well with a team most of the time with clear communication
+    * E = Working well with a team all time with clear and constructive
+      communication
 * Understanding and avoiding red flags
+  * N = Ignoring all red flags
```

Comment:
Where do you find what is a red flag? Is that important here? This section is confusing.

Reply:
This is a good point! If you check the "Philosophy of Software Design" book you can see some examples of these red flags. Perhaps some examples are needed in the assessment guide?

Reply:
Me and my row are working on this change.

Reply:
This suggestion has been addressed in a new pull request from @leonardoz15